brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1072 - improvement: Update new SVGs for container, Image, Multi-buttons child and Taxonomy block.

### Description

- Updated SVGs in a container, Image, Multi-buttons child, and Taxonomy block.

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop block is mentioned in the description.
- Apply box shadow by selecting Presets.
- Check whether it is reflected properly in the editor or not.
- Some test cases in the card also.

### Checklist:
